ara_plumes/regressions.py

```diff
@@ -85,9 +86,9 @@ def do_polynomial_regression(
 
 
 def do_sinusoid_regression(
-    X: Float1D,
+    X: Float2D,
     Y: Float1D,
-    initial_guess: tuple[float, float, float, float] = (1, 1, 1, 1),
+    initial_guess: tuple[float, float, float, float],
```

Comment:
Good call!  There's rarely going to be a good default argument here, so it makes sense to remove it and put that responsibility on the caller.